phantom-dsl/src/main/scala/com/outworkers/phantom/ops/QueryContext.scala

```diff
@@ -396,6 +397,8 @@ abstract class QueryContext[P[_], F[_], Timeout](
     override def compare(x: Option[T], y: Option[T]): Int = {
       { for (x1 <- x; y1 <- y) yield ev.compare(x1, y1) } getOrElse 0
     }
+
+    def parseString(str: String): Option[Option[T]] = ???
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Usage of ??? operator](https://app.codacy.com/manual/flavian/phantom/pullRequest?prid=3695041)